.github/workflows/ci_cd.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.11"
+          python-version-file: "pyproject.toml"
```

Comment:
The `jobs.package` in theory can run on any Python version (also the latest one) since it does not run any code related to the JaxSim package. The version stored in `pyproject.toml` is instead related to the runtime version required by JaxSim.

I don't see any problem in reading it here (it would work regardless) but it has in the same time no real advantage. This job should be successful also using Python versions older than `pyproject.toml@project.requires-python`.

---

- https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-file-input
- https://github.com/ami-iit/jaxsim/actions/runs/9858438537/job/27219807789?pr=197#step:3:9

Reply:
Alright! Do you want me to drop that commit?

Reply:
I don't have a strong preference, what do you think? Probably having it explicit as before may be more clear as it does not raise the doubt of which version gets installed?

Reply:
Yes it makes sense!

Reply:
Done with rebase https://github.com/ami-iit/jaxsim/compare/c6b9cea8232746180b43fb99acf702a3baf4d995..b79406c2f8805a8585039387d07e754e1333bafa